FileSystem/.clasp.json

```diff
@@ -0,0 +1 @@
+{"scriptId":"1xfp5m9qRKa51cImpXHogaOkAauJNu9nO0_MQbqpciqCGflYhU9nyhn1S"}
```

Comment:
Not sure if we want to add this in github since this will be different for each user?

Reply:
Removed the file.